app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  has_many :posts
```

Comment:
- [ ] Please ensure to include setting the foreign key for example `has_many :posts, foreign_key: 'author_id'`
- [ ] in the User class the has_many side of the one-to-many relationship between user and like and user and comment should be set as well, make sure to include the foreign as mentioned for the posts.